test/Microsoft.OpenAPI.OData.Reader.Tests/PathItem/MediaEntityPathItemHandlerTests.cs

```diff
@@ -180,11 +211,22 @@ private IEdmModel GetEdmModel(string annotation)
         <Property Name=""Logo"" Type=""Edm.Stream""/>
         <Property Name = ""Description"" Type = ""Edm.String"" />
          </EntityType>
-      <EntityContainer Name =""TodoService"">
-         <EntitySet Name=""Todos"" EntityType=""microsoft.graph.Todo"" />
+      <EntityType Name=""user"" OpenType=""true"">
+        <NavigationProperty Name = ""photo"" Type = ""microsoft.graph.profilePhoto"" ContainsTarget = ""true"" />
+      </EntityType>
+      <EntityType Name=""profilePhoto"" HasStream=""true"">
+        <Property Name = ""height"" Type = ""Edm.Int32"" />
+        <Property Name = ""width"" Type = ""Edm.Int32"" />
+      </EntityType >
+      <EntityContainer Name =""GraphService"">
+        <EntitySet Name=""Todos"" EntityType=""microsoft.graph.Todo"" />
+        <Singleton Name=""me"" Type=""microsoft.graph.user"" />
       </EntityContainer>
-      <Annotations Target=""microsoft.graph.TodoService/Todos"">
-        {0}
+      <Annotations Target=""microsoft.graph.GraphService/Todos"" >
+       {0}
+      </Annotations>
+      <Annotations Target=""microsoft.graph.GraphService/me"" >
+       {0}
       </Annotations>
```

Comment:
`AcceptableMediaTypes` can not be applied to `EnitySet` or `Signleton`, it can only be applied to `EntityType`, `Property` or `Term` - https://github.com/oasis-tcs/odata-vocabularies/blob/61542d91d01a8bade005317d922270011a89a0a1/vocabularies/Org.OData.Core.V1.xml#L389.
> Lists the MIME types acceptable for the annotated entity type marked with HasStream="true" or the annotated binary, stream, or string property or term

Reply:
Noted.
Though the annotations defined here are for the [ReadRestrictions ](https://github.com/microsoftgraph/OpenAPI.NET.OData/blob/1c1f89520bdbb8a68f1e43a04c55de003f418787/test/Microsoft.OpenAPI.OData.Reader.Tests/PathItem/MediaEntityPathItemHandlerTests.cs#L105-L114) and [UpdateRestrictions](https://github.com/microsoftgraph/OpenAPI.NET.OData/blob/1c1f89520bdbb8a68f1e43a04c55de003f418787/test/Microsoft.OpenAPI.OData.Reader.Tests/PathItem/MediaEntityPathItemHandlerTests.cs#L127-L132).